SUMMARY.md

```diff
@@ -1433,7 +1432,7 @@
       * [Ensure the GKE metadata server is enabled](reference/recipes/terraform/gcp/ensurethegkemetadataserverisenabled.md)
     * [Search](reference/recipes/terraform/search/README.md)
       * [Find Terraform resource](reference/recipes/terraform/search/findresource.md)
-  * [Core](reference/recipes/core-README.md)
+  * [Core](reference/recipes/core-readme.md)
```

Comment:
Why are we changing the capitalization here? Was it incorrect before?

Reply:
My IDE gave me a warning on this link, and it does appear to be lower case:
https://github.com/openrewrite/rewrite-docs/blob/master/reference/recipes/core-readme.md
On other projects we try not to commit if the IDE has any warning, hence why I changed this now.

Reply:
Cool - thanks for taking care of both of those issues then!